lib/launchers/openstack10.rb

```diff
@@ -686,6 +710,54 @@ def assign_ip(instance)
         raise "could not associate a floating ip:\n#{res[:response]}"
       end
     end
+    
+    def allocate_floating_ip(network_name, reuse=true)
```

Comment:
better to use `reuse: true`. Then when you call the method it will be `allocate_floating_ip("my-network-name", reuse: false)` for example. In this way reader knows easier what this second parameter means.

Reply:
make sense